department-of-veterans-affairs/va.gov-cms
# Prep code for PHP 7.4 release

## Description

Drupal updates to work with PHP 7.4.  This includes a Drupal core update.  The servers will still be on php 7.1.  This PR makes the code compatible with 7.4 when the servers are upgraded.

## Testing done

- Run yaml-tests
- Edit and save nodes on the site.

QA Steps 
- [x] Visit /admin/reports/status and validate that the page loads.
- [x] Paragraph Validation:
* Edit Node https://va-gov-cms.lndo.site/node/2727/edit
* Update the `WYSIWYG` within the "Main Content" to `Connect with a care coordinator, They are great`

![Edit Detail Page Returning service member care | VA gov CMS 2020-03-30 10-35-47](https://user-images.githubusercontent.com/121603/77925997-8ff19080-7273-11ea-9ba3-c681daf61162.png)

* Save the Node
* Verify the title was updated

![VA Wilmington Health Care | Returning service member care 2020-03-30 10-36-19](https://user-images.githubusercontent.com/121603/77926028-997af880-7273-11ea-8bf0-e4e2256e8c46.png)


- [x] Build vets-website, any errors will occur in the build validaiton steps.  Below is the command:

```yarn run watch -- --drupal-address=http://va-gov-cms.lndo.site --drupal-user=axcsd452ksey --pull-drupal```

## Definition of Done


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/1280)
